main/adapters/repository/database.go

```diff
@@ -328,7 +352,7 @@ func (dm *DatabaseManager) isRecoverable(err error) bool {
 				return true
 			}
 			if liteErr.Code() == 1 { // SQLITE_ERROR
-				err = dm.CreateTable(SQLiteIdentity)
+				err = dm.CreateTables(createSQLite)
```

Comment:
Well to put it more concise, my concern is a): that depending on how the database behaves, creating a new table might overwrite or clear data that is already there. Also I was wondering if there is maybe a specific way to test for non-existing tables. I.e. error 1 -> check for 'no table' -> yes: create table no: error.  Because my concern b) is hard-to-debug side effect if this gets triggered by some 'random' general errors which might be given by our current or some future database in other caes than 'no table'.

Reply:
Result of discussion: we will make requirements for the create table statements more clear by comments, they need to be able to be safely called at any time.

Reply:
- https://github.com/ubirch/ubirch-client-go/pull/103/commits/acd0df00aaf01d9d984529a5b5e30f4e2bd9c7d8
- https://github.com/ubirch/ubirch-client-go/pull/103/commits/e02c55854cfa0060be023d4bedd9c8f97bf3ca95